web/src/components/RunCard/RunCard.tsx

```diff
@@ -24,17 +25,18 @@ function getIcon(state: TTestRun['state'], failedAssertions: number) {
   return <S.IconSuccess />;
 }
 
-const ResultCard = ({
+const RunCard = ({
```

Comment:
Are we reusing this card for the moment or do you think we'll keep the same one after we integrate the BE services?